illuminate/database/2020-08-06.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Guard bypass in Eloquent models
+link:      https://blog.laravel.com/security-release-laravel-61834-7232
+cve:       ~
+branches:
+    5.5:
```

Comment:
Needs to be quoted (`"5.5:"`), same below

Reply:
actually, `"5.5":`

Reply:
@fabpot Fixed it.

Note to myself: check CI status for errors when doing a PR